source/scripting/VehicleHash.cs

```diff
@@ -179,7 +179,7 @@ public enum VehicleHash : uint
 		Felon2 = 4205676014u,
 		Feltzer2 = 2299640309u,
 		Feltzer3 = 2728226064u,
-		FireTruck = 1938952078u,
+		FireTruk = 1938952078u, // exact name of firetruck (except for case)
```

Comment:
@Tanjitsu @kagikn Why, though? I mean, yeah, it's the correct model name, but the enum doesn't follow the model name convention anyway. Cleaning up some spelling only makes sense in my eyes. (Continuing discussion from #682)

Reply:
Yes, we don't need to follow the model name convention (that Rockstar use) always.

Reply:
True... but being the correct model name would make it more versatile.
If you stored the string, you can call the model without needing to convert it.

All the other Enum's are loadable using the string name rather than the uint.
Except "FireTruck". I went through the list and tested string V's uint.

Reply:
Actually, not all ped and vehicle enum can't be used as string model name (e.g. `Agent ` is originally named `ig_agent`, but you can't load the `ig_agent` model with `Agent`).

Reply:
Sorry i should have specified Vehicle Enums.

Reply:
The main reason for following the names R* uses is to make updating this easier. So rather than updating it by hand, you could just dump the new models from openiv and have a script generate the new entries

Reply:
Yeah having a script generate the new entries would be cool, but I have no idea how to implement it.